packages/itwinui-css/package.json

```diff
@@ -49,10 +49,11 @@
     "build": "yarn clean && yarn build:css && node ../shared/copyrightLinter.js --fix css/* && yarn build:html",
     "build:css": "node ./scripts/generateCss.js src css",
     "build:html": "vite build backstop/tests --config vite.config.ts",
-    "build:watch:scss": "sass --watch src/all.scss css/all.css",
+    "build:watch:scss": "sass --watch src/all.scss:css/all.css src/global.scss:css/global.css",
```

Comment:
Reopening a tangential thread here.

I just realized (as I was doing the monorepo merge) that all our react components rely on individual imports, so we *will* need to build all files if we want the react components to hot reload when changing a css file.

This can be a separate PR/issue or you could incorporate it into the same one because it would allow you to use `global.css` and `anchor.css` in index.

Reply:
Now the `dev` script runs `yarn build:css` first that builds all the individual CSS files. I reverted `index.html` to import `global.css` + `anchor.css`

Reply:
this will not fully fix the problem. if any of the scss files change, it will not get rebuilt.

we want to incrementally compile all files (e.g. `sass --watch`) so that making a change in one of the `scss` files causes storybook (for example) to hot reload.

Reply:
I tried using something similar to `sass --watch` but couldn't come up with a proper command. Because the scss files are in subfolders (e.g. `src/alert/alert.scss`, `src/anchor/anchor.scss`) but we want the output CSS files to be directly in the CSS folder (e.g. `css/alert.css`, `css/anchor.css`)

```
sass src:css --watch
```

does this:
* `Compiled src\alert\alert.scss to css\alert\alert.css`
* `Compiled src\anchor\anchor.scss to css\anchor\anchor.css`
* ...

It also transforms the `classes.scss` and `index.scss` files. I guess that is why we were using the `scripts/generateCss.js` script and only sass watching the `all.scss`.

Also, not sure, but I think the `sass` command doesn't accept blob paths.

What command should I use?

Reply:
My first thought was to use a node [`child_process`](https://nodejs.org/api/child_process.html) for every file, but i think that is actually unnecessary. Sass already accepts multiple files as inputs and lets us specify the output path for each. So all we really need to do is generate those arguments programmatically and run that single command through `child_process`.

I've created https://github.com/iTwin/iTwinUI/pull/935 with that approach. Can you test if it does what you expect?